include/ensmallen_bits/nsga2/nsga2.hpp

```diff
@@ -180,7 +180,7 @@ class NSGA2 {
    * @tparam MatType Type of matrix to optimize.
    * @param population The elite population.
    * @param objectives The set of objectives.
-   * @param calculatedObjectives Vector to store calculated objectives.
+   * @param populationFval Vector to store calculated objectives.
```

Comment:
I'm not sure I see the reason to change the name of the parameter here, but I don't mind to change it, but let's choose a more descriptive name here, I mean `population` is clear, but `Fval` is not really a parameter name that describes what the parameter is about. Let me know what you think.

Reply:
My reasoning is ```calculatedObjectives``` is too vague and lengthy on top of it. 

_"calculatedObjectives of what?"_  A user might ask.

Perhaps ```Fval``` is bad naming, how about ```populationFitness```?